debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/PostgresConnectorIT.java

```diff
@@ -1294,7 +1294,7 @@ public void shouldUpdateReplicaIdentityWithOneTable() throws Exception {
         }
     }
 
-    @Ignore("YB Note: alter replica identity unsupported, see https://github.com/yugabyte/yugabyte-db/issues/21599")
+    @Ignore("YB Note: alter replica identity INDEX is unsupported")
```

Comment:
Just the syntax is supported.

Reply:
Looks like just the syntax is also not supported:
```
Caused by: com.yugabyte.util.PSQLException: ERROR: ALTER TABLE REPLICA IDENTITY USING INDEX not supported yet
  Hint: See https://github.com/yugabyte/yugabyte-db/issues/1124. React with thumbs up to raise its priority
```